content/03.ml-concepts.md

```diff
@@ -1,2 +1,14 @@
 ## Rule 1: Concepts that apply to machine learning also apply to deep learning
+Deep learning is a distinct subfield of machine learning, but it is still a subfield.
+Deep learning has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches, but it is not immune to the many limitations inherent to machine learning.
+Many best practices for machine learning apply to deep learning as well.
+For instance, deep supervised learning models should be trained, tuned, and tested on non-overlapping datasets.
+Those developing deep learning models should first investigate their data.
+When coupled with imprudence, data that is biased, skewed, or of low quality will produce models of dubious performance and limited generalizability.
```

Comment:
I would also add "non-salient information" or sth like this. Because data can be high-quality, non-biased, non-skewed but if the information is not useful for the problem at hand, it may still result in poor models

Reply:
@rasbt I believe that this has been addressed with the most recent commit.